### PR TITLE
Feature/LJI-28-configure-postgres

### PR DESCRIPTION
**Pull request by NormantasN**
This PR introduces the following features:
1. Different profiles to run the project:
With docker-compose and other .yml configurations we can now run the application in different profiles - development (PostgreSQL) and production (H2) simultaneously on different ports or separately.
2. Made corrections to the changelogs. Since H2 is using random_uuid() and postgres is using gen_random_uuid(), each database now has its own folder for the table creations.

### CODE REVIEW

File: src/main/resources/application-dev.yml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    contexts: h2
+  h2:
+    console:
+      enabled: true
+      path: /h2-console
+  data:
+    web:
+      pageable:
+        default-page-size: 10
+server:
```
Comment (SavaLX) [inline src/main/resources/application-dev.yml:27]: It seems like you can move it to the global settings:

```
server:
  error:
    include-message: always
    include-binding-errors: always
    port: 8080
```

Comment (NormantasN) [reply src/main/resources/application-dev.yml]: Done

**Review by SavaLX — APPROVED** [review]

File: src/main/resources/application-dev.yml [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    driverClassName: org.h2.Driver
+    username: sa
+    password: ${DB_PASSWORD}
+  jpa:
+    database-platform: org.hibernate.dialect.H2Dialect
+    show-sql: true
+    hibernate:
+      ddl-auto: none
+  liquibase:
+    change-log: classpath:changelog/changelog-master.xml
```
Comment (SavaLX) [inline src/main/resources/application-dev.yml:17]: As we have one master file for the both profiles, than it's also can be moved to the main property file

File: src/main/resources/changelog/dml/5_insert_plans_table.xml [hunk 3]
```diff
@@ -8,11 +8,11 @@
     <changeSet id="4" author="NormantasN">
         <comment>Insert sample data into plans table</comment>
         <sql>
-          INSERT INTO PLAN_ENTITY (plan_name, status, plan_type, start_date, end_date)
+          INSERT INTO plan (plan_name, status, plan_type, start_date, end_date)
```
Comment (SavaLX) [inline src/main/resources/changelog/dml/5_insert_plans_table.xml:11]: As we mentioned with Anton at the start of our training, don't edit the created changelogs, you have to create new changelogs to edit your database

**Review by SavaLX — CHANGES_REQUESTED** [review]

File: src/main/resources/application-dev.yml [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+    contexts: h2
+  h2:
+    console:
+      enabled: true
+      path: /h2-console
+  data:
```
[the remark this answers is not in the document]
Comment (NormantasN) [reply src/main/resources/application-dev.yml]: Done

File: src/main/resources/application-dev.yml (hunk 2 above)
Comment (NormantasN) [reply src/main/resources/application-dev.yml]: Done

File: src/main/resources/changelog/dml/5_insert_plans_table.xml (hunk 3 above)
Comment (NormantasN) [reply src/main/resources/changelog/dml/5_insert_plans_table.xml:11]: Done

**Review by Cioperis — APPROVED** [review]

**Comment by SavaLX** [comment]
The same as in this PR https://github.com/Java-POT-Summer-24/pot-java-summer-24/pull/35#issuecomment-2232614725

**Comment by NormantasN** [comment]
> The same as in this PR [#35 (comment)](https://github.com/Java-POT-Summer-24/pot-java-summer-24/pull/35#issuecomment-2232614725)

Pinged a question

File: src/main/resources/application.yml [hunk 5]
```diff
@@ -1,55 +1,16 @@
...
-#    properties:
-#      hibernate:
-#        dialect: org.hibernate.dialect.PostgreSQLDialect
-#  liquibase:
-#    change-log: classpath:changelog/changelog-master.xml
-#  server:
-#    port: 8080
-
-
+    port: 8080
```
Comment (SavaLX) [inline src/main/resources/application.yml:16]: I guess should be on the same level as error

Comment (NormantasN) [reply src/main/resources/application.yml]: Done

**Review by SavaLX — APPROVED** [review]

**Pull request merged**